src/main/java/ch/njol/unofficialmonumentamod/mixins/ThreadExecutorMixin.java

```diff
@@ -18,7 +18,7 @@
 	@Inject(method = "executeTask", at = @At(value = "INVOKE", target = "Lorg/slf4j/Logger;error(Lorg/slf4j/Marker;Ljava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)V", remap = false), locals = LocalCapture.CAPTURE_FAILSOFT, cancellable = true)
 	private void silenceTeamFatal(R task, CallbackInfo ci, Exception exception) {
 		if (UnofficialMonumentaModClient.options.silenceTeamErrors) {
-			if ((Objects.equals(exception.getMessage(), "Player is either on another team or not on any team. Cannot remove from team 'players'.") || Objects.equals(exception.getMessage(), "Cannot invoke \"net.minecraft.scoreboard.Team.getName()\" because \"team\" is null"))) {
+			if (((exception.getMessage().matches("Player is either on another team or not on any team. Cannot remove from team '\\w+'\\.")|| Objects.equals(exception.getMessage(), "Cannot invoke \"net.minecraft.scoreboard.Team.getName()\" because \"team\" is null")))) {
```

Comment:
Exception messages can be null in general - I would add a null check. Also, you can precompile the regex for increased performance.

Reply:
aight